new-client/src/plugins/Collector/CollectorModel.js

```diff
@@ -26,6 +26,10 @@ class CollectorModel {
     this.filty = false;
     this.saving = false;
     this.geometryName = "geom";
+    this.wkt =
+      typeof settings.options.wkt === "undefined"
+        ? false
+        : settings.options.wkt;
```

Comment:
I would suggest the nullish coalescing operator here instead (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). (Just a suggestion, everything will work fine as is).

Reply:
I switched to using the nullish operator, thank you!